### PR TITLE
#162948801 Create a loading component

### PR DESCRIPTION
**Pull request by d-kahara**
#### What does this PR do?
Create a reusable loader component

#### Description of Task to be completed?
Generate a component that will be used throughout the app to show when app is loading data from the server or when performing an operation.

####Screenshots
screenshot 2019-02-17 at 16 46 29

### CODE REVIEW

File: src/components/Loader/index.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const Loader = () => (
+  <div className='loader' id='container'>
+    <div className='loader-double-ring'>
+      <div  />
+      <div  />
+    </div>
+  </div>
+
+
```
Comment (andrewhingah) [inline src/components/Loader/index.jsx]: Remove these blank lines.

File: src/components/Loader/Loader.test.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import Loader from './index';
+
+describe('Loader', () => {
+
+  it('should be defined', () => {
```
Comment (cosmas28) [inline src/components/Loader/Loader.test.jsx:7]: Kindly remove this test and use snapshot testing. We need to hit 100%.

**Review by cosmas28 — CHANGES_REQUESTED** [review]

**Comment by cosmas28** [comment]
Any reason for not having a Feature story for this PR?

**Comment by d-kahara** [comment]
> Any reason for not having a Feature story for this PR?

Added this as a task under the signup feature story

**Review by cosmas28 — APPROVED** [review]

**Pull request merged**